test/properties/custom_propertis/custom_property_anonymous_builder_test.dart

```diff
@@ -45,6 +46,7 @@ void main() {
     });
 
     testWidgets('can change the property', (tester) async {
+      tester.setScreenSize(const Size(2000, 1000));
```

Comment:
Is it the `enterText` that fails without this, or something overflowing? Maybe `ensureVisible` could be used?

Reply:
tbh, I am not completely sure, I didn't end up to check on this specific detail.

Let me know if you think we should be 100% on the cause here.

Reply:
Maybe not worth spending more time on it